src/main/java/org/broadinstitute/consent/http/resources/StudyResource.java

```diff
@@ -148,7 +148,7 @@ public Response deleteStudyById(@Auth AuthUser authUser, @PathParam("studyId") I
         throw new NotFoundException("Study not found");
       }
 
-      boolean deletable = study.getDatasets()
+      boolean deletable = (study.getDatasets() == null || study.getDatasets().isEmpty()) || study.getDatasets()
```

Comment:
Just double checking the intent here: if the study is null or empty, then deletable is true (implying that the study is deletable). Is this correct?